src/sdk/pynni/nni/analysis_utils/sensitivity/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
if `analysis_utils` is only used by model compression, suggest to put this directory under `src/sdk/pynni/nni/compression`